tools/depends/target/libass/Makefile

```
@@ -13,7 +13,8 @@ endif
 
 # configuration settings
 CONFIGURE=cp -f $(CONFIG_SUB) $(CONFIG_GUESS) .; \
-          ./configure --prefix=$(PREFIX)
+          ./configure --prefix=$(PREFIX) \
+                      --enable-shared=no
```

Is there any reason to not use the same argument, which is used for all other autotools dependencies?

```suggestion
                      --disable-shared
```

---

It didn't work. I'll double check, but for some reason the disable didn't work when I was testing it

---

Nope, you are right. Not sure what i came across earlier. Updated.